supla-common/proto.h

```diff
@@ -499,24 +518,48 @@ extern char sproto_tag[SUPLA_TAG_SIZE];
 #define SUPLA_MFR_COMELIT 14
 #define SUPLA_MFR_POLIER 15
 
+// BIT map definition for TDS_SuplaRegisterDevice_*::Flags (32 bit)
 #define SUPLA_DEVICE_FLAG_CALCFG_ENTER_CFG_MODE 0x0010  // ver. >= 17
-#define SUPLA_DEVICE_FLAG_SLEEP_MODE_ENABLED 0x0020     // ver. >= 18
-
+#define SUPLA_DEVICE_FLAG_SLEEP_MODE_ENABLED    0x0020  // ver. >= 18
+#define SUPLA_DEVICE_FLAG_CALCFG_SET_TIME       0x0040  // ver. >= 20
+
+// BIT map definition for TDS_SuplaRegisterDevice_F::ConfigFields (64 bit)
+// type: TDeviceConfig_StatusLed
+#define SUPLA_DEVICE_CONFIG_FIELD_STATUS_LED            (1ULL << 0)  // v. >= 20
```

Comment:
I don't know how ULL will behave on different platforms where unsigned long long has different size.

Reply:
Accorging to: https://en.cppreference.com/w/cpp/language/types
ull is at least 64 bits.
Also in C99 ULL is at least 64 bits.